web/app/Style.js

```diff
@@ -77,5 +77,7 @@ Ext.define('Traccar.Style', {
     coordinatePrecision: 6,
     numberPrecision: 2,
 
-    reportTagfieldWidth: 375
+    reportTagfieldWidth: 375,
+
+    headerButtonsMargin: '0 5'
```

Comment:
Five is left margin, without this it look like:
![image](https://cloud.githubusercontent.com/assets/5688080/19794823/043f14ac-9cef-11e6-9424-c1d3fc3c4e59.png)